enclave/core/cwd.c

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/corelibc/errno.h>
+#include <openenclave/corelibc/limits.h>
+#include <openenclave/corelibc/pthread.h>
+#include <openenclave/corelibc/stdio.h>
+#include <openenclave/corelibc/stdlib.h>
+#include <openenclave/corelibc/string.h>
+#include <openenclave/corelibc/sys/stat.h>
+
+static char _cwd[OE_PATH_MAX] = "/";
```

Comment:
Do you think by default this should be initialized to the cwd of the host?

Reply:
Oh I see, this is abstracted out by the device that gets mounted to a specific path, forget my comment above. Interesting!